app/index.html

```diff
@@ -24,6 +24,14 @@
         <script src="bower_components/modernizr/modernizr.js"></script>
         <script src="bower_components/responsive-nav/responsive-nav.js"></script>
         <!-- endbuild -->
+        <script>
```

Comment:
I moved it here as per [Google's suggestion](https://support.google.com/analytics/answer/1008080?hl=en), It's asynchronous so will not impact page loading.
> Paste your snippet (unaltered, in its entirety) into every web page you want to track. Paste it immediately before the closing </head> tag.